sources/packages/backend/apps/api/src/route-controllers/application-restriction-bypass/_tests_/e2e/application-restriction-bypass.aest.controller.getApplicationRestrictionBypasses.e2e-spec.ts

```diff
@@ -0,0 +1,119 @@
+import { HttpStatus, INestApplication } from "@nestjs/common";
+import {
+  E2EDataSources,
+  RestrictionCode,
+  createE2EDataSources,
+  createFakeUser,
+  saveFakeApplication,
+  saveFakeApplicationRestrictionBypass,
+} from "@sims/test-utils";
+import {
+  AESTGroups,
+  BEARER_AUTH_TYPE,
+  createTestingAppModule,
+  getAESTToken,
+} from "../../../../testHelpers";
+import * as request from "supertest";
+import {
+  ApplicationStatus,
+  OfferingIntensity,
+  RestrictionActionType,
+  User,
+} from "@sims/sims-db";
+
+describe("ApplicationRestrictionBypassAESTController(e2e)-getApplicationRestrictionBypasses.", () => {
+  let app: INestApplication;
+  let db: E2EDataSources;
+  let sharedMinistryUser: User;
+
+  beforeAll(async () => {
+    const { nestApplication, dataSource } = await createTestingAppModule();
+    app = nestApplication;
+    db = createE2EDataSources(dataSource);
+    sharedMinistryUser = await db.user.save(createFakeUser());
+  });
+
+  it("Should not get a list of application restriction bypasses for a draft part-time application when there is no restriction bypass associated with it.", async () => {
```

Comment:
I am not sure if we want to keep the last part `when there is no restriction bypass associated with it.` as a part of the test description because it is redundant as draft application can never have restrictions and bypasses.